content/nim/fundamentals/tech-specs.md

```diff
@@ -83,6 +83,29 @@ The following benchmarks focus on **disk storage** requirements for NGINX Instan
 - **Medium configuration**: Usually includes about **50 servers**, **200 locations**, and **200 upstreams/backends**. Each instance generates **16,766 metrics per minute**.
 - **Generic Large configuration**: Handles up to **100 servers**, **1,000 locations**, and **900 upstreams/backends**. In **NGINX Plus**, each instance generates **59,484 metrics per minute**.
 
+
+#### Directory Requirements for NGINX Instance Manager
+
+Below are the directory requirements and storage recommendations for fresh, minimal, and moderate usage (<100 instances) of NGINX Instance Manager (NIM).
+
+Important Note: These recommendations apply if you are using NGINX Agent to connect NGINX instances to NIM for full management capabilities (e.g., managing configurations, viewing metrics, applying WAF policies, managing certificates, etc.). If the use case is solely usage reporting, as introduced in R33+, the Agent is not required, and resources needed are significantly reduced. For usage reporting-only deployments, NIM simply receives and stores usage data sent directly from the instances, which does not require the same resource allocation outlined below.
+
+- /usr/bin: Stores NIM binaries; recommend 500MB (current size ~400MB).
+- /var/lib/nms/dqlite: Stores DQLite database data; recommend 1GB (2GB without NAP, 5GB with NAP enabled and large compiled bundles).
+- /var/lib/nms/streaming: Stores NATS streaming messages; recommend 500MB.
+- /var/lib/nms/secrets: Stores secrets for LLM license handshakes; recommend 10MB.
+- /var/lib/nms/modules: Stores static content like manager.json; recommend 100KB (12KB minimum).
+- /var/lib/clickhouse: Stores ClickHouse metrics data; recommend 2.5GB per instance (25GB for 10 instances, 250GB for 100 instances). This is only required if ClickHouse metrics are enabled. In NIM 2.20, lightweight mode can be deployed without ClickHouse (no metrics collected).
```

Comment:
Some of what's drafted is redundant. However, this is begging for a link to answer "What if I'm deploying > 100 instances of NIM"

```suggestion
- /var/lib/clickhouse: Stores ClickHouse metrics data; recommend 2.5GB per instance. Required only if ClickHouse metrics are enabled. In NIM 2.20, you can deploy lightweight mode without ClickHouse.
```